tests/telemetry/test_writer.py

```diff
@@ -36,7 +29,7 @@ def test_add_event(telemetry_writer, test_agent_session):
     assert len(requests) == 1
     assert requests[0]["headers"]["Content-Type"] == "application/json"
     assert requests[0]["headers"]["DD-Telemetry-Request-Type"] == payload_type
-    assert requests[0]["headers"]["DD-Telemetry-API-Version"] == "v1"
+    assert requests[0]["headers"]["DD-Telemetry-API-Version"] == "v2"
```

Comment:
Can we add test coverage for the new headers telemetry headers

Reply:
:+1: